packages/block-editor/src/components/list-view/branch.js

```diff
@@ -186,11 +205,18 @@ function ListViewBranch( props ) {
 								isSyncedBranch={ syncedBranch }
 							/>
 						) }
-						{ ! showBlock && (
-							<tr>
-								<td className="block-editor-list-view-placeholder" />
-							</tr>
-						) }
+						{ ! showBlock &&
+							( isSelected ? (
+								<PlaceholderRow
+									clientId={ clientId }
+									isSelected={ isSelected }
+									selectedClientIds={ selectedClientIds }
+								/>
+							) : (
+								<tr>
+									<td className="block-editor-list-view-placeholder" />
+								</tr>
+							) ) }
```

Comment:
I don't quite follow the reasoning for the ternary here, as the else state seems to render the same thing as `PlaceholderRow`.

Reply:
I wasn't sure if there was a performance reason for using `<tr>` directly over a separate React component for the placeholder rows, so this ternary preserved the `<tr>` for the majority of the placeholders. It'd be neater to remove it and just use `PlaceholderRow`, though, so I'll have a play with consolidating and see how that goes.